markdown/introduction.md

```diff
@@ -5,6 +5,18 @@ Nav: home.nav
 
 TableOfContents
 
+## {id="what"} What does it, really?
+
+What does it mean that "Bedrocklinux allows to run a linux system that is composed of different linux distributions"? It means, the system boots with a bootloader, a kernel, and an init system that can be independently selected from the "bedrocked" distributions, and can then freely run userland packages that are installable side-by-side from the different distributions.
+
+Consequently, the running system is rooted in different linux distributions.
+Therefore, a descriptive name for bedrocklinux concept could actually be multi-rooted-linux.
+
+In a pretentious language it could also have been marketed as something like "distrosnip", because, in a way, a multi-rooted system takes to an end what appimages, flatpacks or snap packages had begun.
+
+However, instead of shipping static binary collections like the appimages, flatpacks and snap packages (the last two introducing third-party appstores) a multi-rooted system can avoid installing redundant, often outdated and not well maintained binary collections. In a multi-rooted-linux system the different roots are managed by the original distribution's package managers and security update mechanisms, and a tool called *pmm*, the "package manager manager",  allows to manage the different roots in a unified way. While sandboxing can be done by using standard tools like firejail in the roots.
```

Comment:
The idea behind Bedrock is to get features from other distros.  It doesn't really speak to the problem of developers trying to make packages for the plethora of distros out there.

The idea behind appimage, flatpak, and snap is to make distro-agnostic packages.  These don't really speak to the problem of getting software already made for other distros onto your system.

Bedrock isn't a "better" appimage/flatpak/snap.  It's largely orthogonal to them.

Reply:
I had been making some edits before you commented, and with them I may kind of realized that orthonogality between "distributing vs. collecting software" better and had put the word "meta-collection" in the index.md.

How do you like meta-collection, or meta-collection tool instead of meta-distribution?

Reply:
I still think bedrock could be a preferable solution compared to new and independent only-package-once projects to run software everywhere.

I mean the guix and nix package managers can also make their trees run within other distros, but bedrock more generally allows to run the original distro trees (including package managers) not just within another distro but within a customized system (a user's meta-collection).

Reply:
I prefer meta-distribution.  "meta-collection" / "meta-collection tool" are ambiguous as to what they are collections _of_.

Most people who run into Bedrock already know what a Linux distribution is.  While the notion of a _meta_ distribution is new, the fact it's rooted in the preexisting well established "Linux distribution" concept is advantageous; we only have to explain the "meta" part.  Moving to "meta-collection" is a regressive move.

Reply:
Note, I don't consider the flatpack and snap store-approach to be suitable solutions for free and open source software, only guix and nix, but bedrock can make any existing, well maintainable packages run everywhere.

Reply:
What I wanted with this paragraph is to provide a broader picture and some connections to related topics.
Let me try to take that orthogonality and similarity into account:
```suggestion
Compared to making packages portable by shipping mostly static binary collections like appimages, flatpacks and snap packages (the last two even slipping in third-party appstores) a multi-rooted bedrock linux system can often avoid installing redundant, and sometimes outdated binary collections. The different software roots are managed by their original package managers and security update mechanisms (if any), the software in the roots continues to be maintained collectively as good as in their original distribution. A tool called *pmm*, the "package manager manager",  allows to manage the different roots in a unified way. And sandboxing can be done by using standard tools like firejail in the roots.
More generally than nix and guix package managers, bedrock allows to run unmodified legacy distro packages, not only within another distro, but within a fully customizable system, the user's bedrock meta-collection.
```